config-sample.yaml

```diff
@@ -12,6 +12,10 @@ s3:
   AWSAccessKeyID: "AKIAXOMDBDAFLLTWLU4Y"
   AWSSecretAccessKey: "yepL/xP02oCuadGiXnWWkaKF5PP4ORZKKBxNDRDy"
   BucketName: "miner-pictures"
+python:
+  # Change when the microservice host for prompt is available
+  microserviceprompthost: "https://yiowpv.deta.dev/api/microservice/content"
+
```

Comment:
we're not using this functionality again,
is there any reason you included it again.

Reply:
I have removed this from the conflict

Reply:
I know
you should also remove this here 
like it should not be here
because if I merge it'll enter the codebase

Reply:
deleted